client/src/components/register-form.tsx

```
@@ -0,0 +1,122 @@
+      <AuthForm type="register" onSubmit={handleRegisterSubmit} />
+      {isDialogOpen && (
+        <Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
+          <DialogContent>
+            <DialogHeader>
+              <DialogTitle>Verify your email</DialogTitle>
+            </DialogHeader>
+            <form
+              onSubmit={handleOtpSubmit}
+              className="flex flex-col items-center justify-center gap-2"
+            >
+              <label className="text-lg font-semibold">One-Time Password</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)